cheat.php

```diff
@@ -713,6 +770,48 @@ function ExecuteRequest( $Method, $URL, $Data = [] )
 	return $Data;
 }
 
+function GetLocalScriptHash( )
```

Comment:
hash_file

Reply:
Going to use sha1_file instead, since we are only using sha1 anyway.

Reply:
And use `__FILE__`